### DISCUSSION TITLE
最新版模板和overleaf版本的区别问题

### DISCUSSION THREAD

**Discussion by ylxiang2018** [sjtug/SJTUThesis#718] [提问] [answered]
大家好，请问sjtuthesis的overleaf模板和最新版模板有什么区别吗?

我现在直接用的overleaf版模板，想了解一下区别。
没用最新版模板是因为用最新版模板在overleaf上会报这个问题:
Underfull \hbox (badness 2269) in paragraph at lines 59--59。

**Comment by skyzh** [comment, marked as answer]
是否使用了 XeLaTeX 编译？

**Reply by ylxiang2018** [reply]
使用了。其实这个不算错误，应该是个警告吧。正常显示感觉没啥问题。
我只是想了解一下这两个模板区别是啥，如果没啥区别我就直接用overleaf模板了。

**Reply by skyzh** [reply]
Overleaf 上的应该是 0.9 版本，和现在的最新版还是有一些区别的。建议下载最新版 source code 传上去用。

**Reply by ylxiang2018** [reply]
那报这个警告没什么影响吧?
Underfull \hbox (badness 2269) in paragraph at lines 59--59
这个59行是main.tex \input参考文献的地方

**Reply by skyzh** [reply]
应该没有

**Reply by ylxiang2018** [reply]
好的，谢谢

**Reply by zepinglee** [reply]
应该是 Overleaf 上的版本使用了不同的字体，所以最后输出时有略微的差异导致这个地方没填满一行（Underfull \hbox）。

**Reply by ylxiang2018** [reply]
那应该怎么解决呢?
最新版模板导入overleaf会有这个问题。直接用overleaf模板没有。

**Reply by skyzh** [reply]
可能是最新版的 bug，虽然不影响使用，之后会尝试解决一下（

**Reply by ylxiang2018** [reply]
好的，辛苦了
